circuit-foundation/src/androidUnitTest/kotlin/com/slack/circuit/foundation/NavigableCircuitViewModelStateAndroidTest.kt

```diff
@@ -36,13 +37,12 @@ class NavigableCircuitViewModelStateAndroidTest {
   @Test
   fun retainedStateScopedToBackstackWithRecreations() {
     composeTestRule.run {
-      mainClock.autoAdvance = false
+      mainClock.autoAdvance = true
 
       // Current: Screen A. Increase count to 1
       onNodeWithTag(TAG_LABEL).assertTextEquals("A")
       onNodeWithTag(TAG_COUNT).assertTextEquals("0")
       onNodeWithTag(TAG_INCREASE_COUNT).performClick()
-      mainClock.advanceTimeByFrame()
```

Comment:
This test had a lot of assumptions about what happens in a single frame. Decided it was easier just to use `autoAdvance = true` for the majority of the test, and handle the `onEachFrameWhileMultipleScreens` calls with `autoAdvance = false`.